libs/testing/src/user.session.ts

```diff
@@ -281,14 +400,19 @@ export class UserSession {
     }
   }
 
+  // TODO: create EE version
   async switchEnvironment(environmentId: string) {
     const environment = await this.environmentService.getEnvironment(environmentId);
 
     if (environment) {
       this.environment = environment;
       await this.testAgent.post(`/v1/auth/environments/${environmentId}/switch`);
 
-      await this.fetchJWT();
+      if (isClerkEnabled()) {
```

Comment:
We will merge this PR after the Env work is released so some adaptation is necessary.